harness/determined/deploy/local/cluster_utils.py

```diff
@@ -24,6 +24,8 @@
 NETWORK_NAME = "determined_default"
 VOLUME_NAME = "determined-db-volume"
 MASTER_NAME = "determined-master_1"
+OSS_VERSION = "determined-master"
```

Comment:
The word `version` is throwing me off (especially when I'm used to versions looking like `0.31.0` or something). Is there a word that matches the use a little more specifically?

Reply:
yeah, I think "Edition" probably works better - I'll make the change

Reply:
Edition looks great!